firebase.ts

```diff
@@ -2,7 +2,7 @@
 import { initializeApp } from "firebase/app";
 // import { getAnalytics } from "firebase/analytics";
 import { getAuth } from "firebase/auth";
-import firebaseConfig from "./.secret/firebase-config.json";
+import { firebaseConfig } from "./.secret/firebase-config"
```

Comment:
we can reverse this line. I changed firebase-config to json when I implemented the github actions workflow